packages/create-toolpad-app/src/index.ts

```diff
@@ -161,6 +159,455 @@ const scaffoldProject = async (absolutePath: string, installFlag: boolean): Prom
+  .lock-wscript
+  
+  # Compiled binary addons (https://nodejs.org/api/addons.html)
+  build/Release
+  
+  # Dependency directories
+  node_modules/
+  jspm_packages/
+  
+  # Snowpack dependency directory (https://snowpack.dev/)
+  web_modules/
+  
+  # TypeScript cache
+  *.tsbuildinfo
+  
+  # Optional npm cache directory
+  .npm
+  
+  # Optional eslint cache
+  .eslintcache
+  
+  # Optional stylelint cache
+  .stylelintcache
+  
+  # Microbundle cache
+  .rpt2_cache/
+  .rts2_cache_cjs/
+  .rts2_cache_es/
+  .rts2_cache_umd/
+  
+  # Optional REPL history
+  .node_repl_history
+  
+  # Output of 'npm pack'
+  *.tgz
+  
+  # Yarn Integrity file
+  .yarn-integrity
+  
+  # dotenv environment variable files
+  .env
+  .env.development.local
+  .env.test.local
+  .env.production.local
+  .env.local
+  
+  # parcel-bundler cache (https://parceljs.org/)
+  .cache
+  .parcel-cache
+  
+  # Next.js build output
+  .next
+  out
+  
+  # Nuxt.js build / generate output
+  .nuxt
+  dist
+  
+  # Gatsby files
+  .cache/
+  # Comment in the public line in if your project uses Gatsby and not Next.js
+  # https://nextjs.org/blog/next-9-1#public-directory-support
+  # public
+  
+  # vuepress build output
+  .vuepress/dist
+  
+  # vuepress v2.x temp and cache directory
+  .temp
+  .cache
+  
+  # Docusaurus cache and generated files
+  .docusaurus
+  
+  # Serverless directories
+  .serverless/
+  
+  # FuseBox cache
+  .fusebox/
+  
+  # DynamoDB Local files
+  .dynamodb/
+  
+  # TernJS port file
+  .tern-port
+  
+  # Stores VSCode versions used for testing VSCode extensions
+  .vscode-test
+  
+  # yarn v2
+  .yarn/cache
+  .yarn/unplugged
+  .yarn/build-state.yml
+  .yarn/install-state.gz
+  `;
+
+  // Define all files to be created up front
+  const files = new Map([
+    ['app/api/auth/[...nextAuth]/route.ts', { content: '' }],
+    ['app/auth/[...path]/page.tsx', { content: '' }],
+    ['app/(dashboard)/page/page.tsx', { content: dashboardPageContent }],
+    ['app/(dashboard)/layout.tsx', { content: dashboardLayoutContent }],
+    ['app/layout.tsx', { content: rootLayoutContent }],
+    ['app/page.tsx', { content: rootPageContent }],
+    ['theme.ts', { content: themeContent }],
+    ['next-env.d.ts', { content: nextTypesContent }],
+    ['next.config.mjs', { content: nextConfigContent }],
+    ['.eslintrc.json', { content: eslintConfigContent }],
+    ['tsconfig.json', { content: tsConfigContent }],
+    ['package.json', { content: JSON.stringify(packageJson, null, 2) }],
+    ['.gitignore', { content: gitignoreTemplate }],
+    // ...
+  ]);
+
+  // Get all directories and deduplicate
+  const dirs = new Set(Array.from(files.keys(), (filePath) => path.dirname(filePath)));
+
+  // Create directories, use recursive option to create parent directories
+  try {
+    await Promise.all(
+      Array.from(dirs, (dirPath) =>
+        fs.mkdir(path.join(absolutePath, dirPath), { recursive: true }),
+      ),
+    );
+  } catch (error: any) {
```

Comment:
You can remove this try/catch. `mkdir` doesn't throw `EEXIST` when `recursive` is `true`
```suggestion
```